app/assets/javascripts/required_fields.coffee

```diff
@@ -165,6 +165,13 @@ $(document).ready ->
         index = id.lastIndexOf(last)
         id = id.slice(0, index)
       label = $("label[for='#{id}']")
+      # if element is a radio button, we want to add the icon to the label
+      # for the group of radio buttons
+      # this was added for UMM-S v1.3.4, but should be evaluated for other types
+      if $(element).is(':radio')
+        $radioParent = $(element).closest('.radio-group-parent')
+        label = $radioParent.siblings('label.radio-group-label')
+
```

Comment:
The required icons for ‘Allow multiple values’ only show up when the user clicks the radio button which means they don’t know the field is required until after they click (which I assume is not the expected behavior). This appears to be caused by the fact that `getLabels` is invoked in the `$optionallyRequiredWithValueFields` loop on 197 in required_fields.coffee

Reply:
This is the expected behavior. The parent object is not required, so those fields are not always required. When the parent field exists, however, it must contain these fields.